dev_requirements.txt

```
@@ -4,8 +4,10 @@ invoke==1.4.1
 mypy==0.761
 pytest==5.3.5
 Sphinx==3.5.4
+jinja2==2.10
```

Can you try this version instead? It is what was working previously
```suggestion
jinja2==2.3
```